assets/css/base/icons.scss

```diff
@@ -8,8 +8,11 @@
 // Sass tools and helpers used across the project.
 @import '../sass/utils/variables';
 @import '../sass/utils/mixins';
+@import '../sass/vendors/font-awesome/brands';
 @import '../sass/vendors/font-awesome/fontawesome';
+@import '../sass/vendors/font-awesome/regular';
 @import '../sass/vendors/font-awesome/solid';
+@import '../sass/vendors/font-awesome/v4-shims';
```

Comment:
Adding in all these other variations increases the size of this file (`icons.css`). When I built, the file grew from 99 KB (`master`) to 144KB on this branch - almost 45% increase.

I'd suggest we keep this focused on the variations we need for Storefront. If we take out `brands` `regular` and `v4-shims` then `icons.css` is 101KB.

Reply:
Sounds good! I'll update this PR later today and will also update Font Awesome to version 5.13 as mentioned in https://github.com/woocommerce/storefront/pull/1279#pullrequestreview-396121519.

Reply:
I've taken care of these updates – thanks for your work on this @nielslange  🤝